src/api.ts

```diff
@@ -55,6 +55,20 @@ export async function myradioApiRequest(
   }
 }
 
+export async function bapsicleApiRequest(
+  endpoint: string,
+  method: "GET" | "POST" | "PUT",
+  params: any
+): Promise<any> {
+  const res = await apiRequest(
+    "http://" + window.location.hostname + ":13500" + endpoint,
```

Comment:
Probably something for the future, but would be nice if this wasn't hardcoded.

Reply:
Mmm, yeah. I'll have a go. When built inside BAPSicle, it's expected to be whatever the port the server is running on anyway, but in separate development it may want to be configurable in the env.